plugins/org.eclipse.n4js/src/org/eclipse/n4js/N4JSGlobals.java

```diff
@@ -17,11 +17,11 @@
 import java.util.Collection;
 
 /**
- * Global hook for static information about the current setup.
- *
- * Start up code must initialize static attributes here.
- *
+ * Global hook for static information about the current setup. Contains file extensions, library names, and other
+ * "useful" strings.
  */
+// TODO avoid GlobalGod class.
+// Break down, specialize, narrow the scope of usage.
```

Comment:
I don't see the problem, here. `N4JSGlobals` contains some string constants and is not even that huge (roughly 2 screen pages).

Reply:
removed